### ISSUE TITLE
hello,I'm confused with your train strategy

### ISSUE THREAD

**Issue by liuxiuxuhaodong** [zhunzhong07/CamStyle#7]
In the paper,you viewed  the real and fake images equally,as to the loss function,  when  defining loss function , why you considered it into two parts ?
